### PR TITLE
Updates to work with TypeScript 1.6

### PR DESCRIPTION
**Pull request by alexgorbatchev**
Would love to see this published. If you want a hand with `gulp-tsfmt`, I'd be happy to help, just add me on github and npm (username alexgorbatchev).

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -52,12 +52,14 @@ class Transformer {
     }
 
     private _applyEdits(file: VinylFile, edits: ts.TextChange[]): Buffer {
-        return edits.reduceRight((contents, edit) => {
-            var head = contents.slice(0, edit.span.start());
-            var tail = contents.slice(edit.span.start() + edit.span.length());
+        const reduce = (contents, edit) => {
```
Comment (alexgorbatchev) [inline src/index.ts:55]: Couldn't figure out how to make linter happy (see https://github.com/palantir/tslint/issues/742)

**Comment by pspeter3** [comment]
Thanks!

**Comment by alexgorbatchev** [comment]
@pspeter3 could you cut a new release please? I suggest a 1.0 release because this has been out for 7 months now (declare it stable and more people will use it).

**Comment by alexgorbatchev** [comment]
@pspeter3 if you want a hand with house keeping, add me on NPM (alexgorbatchev) and here and I'll be happy to help out :)

https://www.npmjs.com/~alexgorbatchev

**Comment by ghost** [comment]
@alexgorbatchev In an issue, @pspeter3 said he gave up on this.

Please fork it and continue.

**Pull request merged**